src/components/ApiErrors/index.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from "react";
+import isEmpty from "lodash/isEmpty";
+import {
+  renderErrorElements,
+  createUsernameAnchor,
+} from "../../utils/renderErrorsUtils";
+import styles from "./ApiErrors.module.css";
+import { ProblemDetail } from "../../interfaces";
+
+interface ApiErrorsProps {
+  problemDetail: ProblemDetail | undefined;
+}
+
+/**
+ * renderApiErrors
+ * This renders the component above the form that displays information on
+ * the error(s) from the submission. Also renders links that link to the
+ * specific input element that is returning an error.
+ */
+const ApiErrors = React.forwardRef<HTMLDivElement, ApiErrorsProps>(
+  ({ problemDetail }, ref) => {
+    if (
+      !problemDetail ||
+      isEmpty(problemDetail) ||
```

Comment:
How would there be an empty problem detail?

Also, this might be easier to read if there were a variable for all the cases in which this should return null...

Reply:
True, there wouldn't be an empty problem detail since the requirement is coming from the type. I can remove that.